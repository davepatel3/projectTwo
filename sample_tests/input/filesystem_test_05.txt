5867
create_directory_root
7Lc_D7BK5nT1-
create_file
52
zloxrH7Vi1GL
1
create_directory
gyfNVdZZsS3fsgT_
1
remove
1
exist
2
create_directory
Fqtc5P
3
print_layout
get_absolute_path
3
create_directory
vS6Qba
0
create_link
5
HFff_Af Ws0L
3
create_directory_root
fHLFqb8nUup27LptQ
get_available_size
create_file
7
UMcMSEBpser1CpTzn8-
1
get_name
6
remove
7
create_directory_root
dkuLq sDNtH_tjf9OD
create_file
41
2a gcw2
5
create_directory
xAziIYWH5C6X1
3
create_file
12
tKV592p0hIwIUA
5
create_file
5
aJx76OSK0o4iqjQfwP
1
create_file
156
hKFuvKUm5
5
get_absolute_path
11
create_directory
HQY1rYoAvQ3QRBsnz8aF
0
create_file
144
NyMHcyR
1
get_file_size_path
/vS6Qba/tKV592p0hIwIUA
create_directory
hyVVKnMWi
4
create_file
8
Mz-I2TP
16
remove
11
create_directory
OYZ1 xVqcUCNqD_8j
4
get_file_size_path
/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Fqtc5P/hyVVKnMWi
create_file
38
y19R97
11
create_directory
8-WhU4JhodkuynZQp
10
create_directory
e-_xuLRMH-
7
print_layout
create_file
69
ZEEX
19
rename
4
Exy5nj0I
exist
19
create_directory
0nIac w0
7
create_directory
Ktd cb
14
get_file_size
8
create_file
9
JmrKjzWNWP
1
create_file
67
kYgDaBuO-HOXiK7EQQle
5
create_file
37
BCE_Of0U_lUvF
7
create_link
11
eUYohP5ZGZ1tQ
7
create_directory
JFACp 3_3yiYbGp1Z
12
create_file
18
dRUy-Mnb-D-7QZzG2
23
print_layout
create_file
134
rLqIFrhT
19
create_link
15
GDUl4czG-C
1
follow
30
create_file
15
qx9xDvMtnxgvMmTVFIHF
14
get_available_size
get_file_size_path
/7Lc_D7BK5nT1-/NyMHcyR
get_name
10
remove
11
get_file_size
12
print_layout
rename
2
LhhToDwzGf
print_layout
get_available_size
create_directory
OOr7e7E_3FSM
1
follow
27
get_absolute_path
11
get_absolute_path
24
create_directory_root
mZeU833d
create_file
16
pOyoLVPd6cXIvE1vJ
11
rename
13
a0esTU
create_directory
7g0fWjdtTOs9YwJ_Zp
5
create_link
22
S3N70hvG-
19
print_layout
create_file
4
f34_3NRc6wUX_LE
32
create_directory
L1Vr_j_U1H u0Sk
7
create_link
20
b2rTnAA8K5 avSI
0
create_link
39
_WfksYZ
35
create_directory
PUeYpHbGH53moIzCC
5
create_file
47
r0vpt6zpnq
23
create_file
90
oqc20qBDZMf
5
get_file_size
6
create_directory_root
bXLdG-ie
follow
6
follow
39
print_layout
get_available_size
print_layout
print_layout
follow
39
create_file
28
NFE6bt
7
rename
8
8rtjmpN732vnMk
create_link
38
l yyApi
23
remove
1
quit
