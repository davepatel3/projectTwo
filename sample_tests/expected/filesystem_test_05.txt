create_directory_root
1
create_file
2
create_directory
3
remove
exist
1
create_directory
4
print_layout
[D]7Lc_D7BK5nT1-
	[F]zloxrH7Vi1GL (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Fqtc5P

get_absolute_path
/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_
create_directory
5
create_link
6
create_directory_root
7
get_available_size
5815
create_file
8
get_name
HFff_Af Ws0L
remove
create_directory_root
7
create_file
9
create_directory
10
create_file
11
create_file
12
create_file
13
get_absolute_path
/vS6Qba/tKV592p0hIwIUA
create_directory
14
create_file
15
get_file_size_path
12
create_directory
16
create_file
17
remove
create_directory
11
get_file_size_path
Invalid handle!
create_file
18
create_directory
19
create_directory
20
print_layout
[D]7Lc_D7BK5nT1-
	[F]zloxrH7Vi1GL (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Fqtc5P
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]hKFuvKUm5 (size = 156)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
[D]HQY1rYoAvQ3QRBsnz8aF

create_file
21
rename
exist
1
create_directory
22
create_directory
23
get_file_size
7
create_file
24
create_file
25
create_file
26
create_link
27
create_directory
Invalid handle!
create_file
28
print_layout
[D]7Lc_D7BK5nT1-
	[F]zloxrH7Vi1GL (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]hKFuvKUm5 (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)

create_file
29
create_link
30
follow
15
create_file
31
get_available_size
5067
get_file_size_path
144
get_name
xAziIYWH5C6X1
remove
get_file_size
5
print_layout
[D]7Lc_D7BK5nT1-
	[F]zloxrH7Vi1GL (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]hKFuvKUm5 (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)

rename
print_layout
[D]7Lc_D7BK5nT1-
	[F]LhhToDwzGf (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]hKFuvKUm5 (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)

get_available_size
5067
create_directory
32
follow
11
get_absolute_path
/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j
get_absolute_path
/7Lc_D7BK5nT1-/JmrKjzWNWP
create_directory_root
33
create_file
34
rename
create_directory
35
create_link
36
print_layout
[D]7Lc_D7BK5nT1-
	[F]LhhToDwzGf (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
				[F]pOyoLVPd6cXIvE1vJ (size = 16)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
				[L]S3N70hvG- [->/dkuLq sDNtH_tjf9OD/0nIac w0]
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
	[D]OOr7e7E_3FSM
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]a0esTU (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
	[D]7g0fWjdtTOs9YwJ_Zp
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)
[D]mZeU833d

create_file
37
create_directory
38
create_link
39
create_link
40
create_directory
41
create_file
42
create_file
43
get_file_size
Invalid handle!
create_directory_root
44
follow
5
follow
20
print_layout
[D]7Lc_D7BK5nT1-
	[F]LhhToDwzGf (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
				[F]pOyoLVPd6cXIvE1vJ (size = 16)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
				[L]S3N70hvG- [->/dkuLq sDNtH_tjf9OD/0nIac w0]
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
	[D]OOr7e7E_3FSM
		[F]f34_3NRc6wUX_LE (size = 4)
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]a0esTU (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
	[D]7g0fWjdtTOs9YwJ_Zp
		[L]_WfksYZ [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
	[D]PUeYpHbGH53moIzCC
	[F]oqc20qBDZMf (size = 90)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
	[D]L1Vr_j_U1H u0Sk
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
		[F]r0vpt6zpnq (size = 47)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)
[D]mZeU833d
[L]b2rTnAA8K5 avSI [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
[D]bXLdG-ie

get_available_size
4910
print_layout
[D]7Lc_D7BK5nT1-
	[F]LhhToDwzGf (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
				[F]pOyoLVPd6cXIvE1vJ (size = 16)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
				[L]S3N70hvG- [->/dkuLq sDNtH_tjf9OD/0nIac w0]
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
	[D]OOr7e7E_3FSM
		[F]f34_3NRc6wUX_LE (size = 4)
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]a0esTU (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
	[D]7g0fWjdtTOs9YwJ_Zp
		[L]_WfksYZ [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
	[D]PUeYpHbGH53moIzCC
	[F]oqc20qBDZMf (size = 90)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
	[D]L1Vr_j_U1H u0Sk
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
		[F]r0vpt6zpnq (size = 47)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)
[D]mZeU833d
[L]b2rTnAA8K5 avSI [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
[D]bXLdG-ie

print_layout
[D]7Lc_D7BK5nT1-
	[F]LhhToDwzGf (size = 52)
	[D]gyfNVdZZsS3fsgT_
		[D]Exy5nj0I
			[D]hyVVKnMWi
				[F]Mz-I2TP (size = 8)
			[D]OYZ1 xVqcUCNqD_8j
				[F]y19R97 (size = 38)
				[F]pOyoLVPd6cXIvE1vJ (size = 16)
		[L]HFff_Af Ws0L [->/vS6Qba]
		[D]xAziIYWH5C6X1
			[D]8-WhU4JhodkuynZQp
				[F]ZEEX (size = 69)
				[F]rLqIFrhT (size = 134)
				[L]S3N70hvG- [->/dkuLq sDNtH_tjf9OD/0nIac w0]
	[F]UMcMSEBpser1CpTzn8- (size = 7)
	[F]aJx76OSK0o4iqjQfwP (size = 5)
	[F]NyMHcyR (size = 144)
	[F]JmrKjzWNWP (size = 9)
	[L]GDUl4czG-C [->/7Lc_D7BK5nT1-/NyMHcyR]
	[D]OOr7e7E_3FSM
		[F]f34_3NRc6wUX_LE (size = 4)
[D]vS6Qba
	[F]2a gcw2 (size = 41)
	[F]a0esTU (size = 156)
	[F]kYgDaBuO-HOXiK7EQQle (size = 67)
	[D]7g0fWjdtTOs9YwJ_Zp
		[L]_WfksYZ [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
	[D]PUeYpHbGH53moIzCC
	[F]oqc20qBDZMf (size = 90)
[D]dkuLq sDNtH_tjf9OD
	[D]e-_xuLRMH-
	[D]0nIac w0
	[F]BCE_Of0U_lUvF (size = 37)
	[L]eUYohP5ZGZ1tQ [->/7Lc_D7BK5nT1-/gyfNVdZZsS3fsgT_/Exy5nj0I/OYZ1 xVqcUCNqD_8j]
	[D]L1Vr_j_U1H u0Sk
[D]HQY1rYoAvQ3QRBsnz8aF
	[D]Ktd cb
		[F]dRUy-Mnb-D-7QZzG2 (size = 18)
		[F]r0vpt6zpnq (size = 47)
	[F]qx9xDvMtnxgvMmTVFIHF (size = 15)
[D]mZeU833d
[L]b2rTnAA8K5 avSI [->/dkuLq sDNtH_tjf9OD/e-_xuLRMH-]
[D]bXLdG-ie

follow
20
create_file
45
rename
create_link
46
remove
quit
